### DISCUSSION TITLE
2007/11/16/%E5%99%A9%E6%A2%A6%E6%88%90%E7%9C%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#190] [Announcements]
# 2007/11/16/%E5%99%A9%E6%A2%A6%E6%88%90%E7%9C%9F

我比较常见的一种噩梦是与牙齿有关的梦。有时候梦见吃东西，吃着吃着，牙齿就松动了，或者碎裂甚至脱落下来。每每这时，我都会极其痛苦。好在梦醒后，可以长出一口气，庆幸这只是个梦。

https://blog.qizhen.xyz/2007/11/16/%E5%99%A9%E6%A2%A6%E6%88%90%E7%9C%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

阿七
2009年11月20日 19:07 编辑
多睡觉，少写程序。

回复
竹贤
2007年11月19日 17:38 编辑
在那个年代，在妈妈肚子里就缺乏营养，缺乏维生素和钙，也就是说先天不足，这是一；婴儿和幼年时代，也是物质贫乏的时代，营养没有跟上，这是二；第三，小时候体质弱，易得病，那时候，常用一种现在淘汰了的药品：四环素和土霉素，这是对儿童的骨骼和牙齿发育十分有害的药品，吃多了，牙齿就不可能好。
现在弥补吧：多吃一点牛奶之类，含钙多，钙又易吸收的食品；保持口腔卫生；常咀嚼口香糖，按摩牙龈，也许对牙齿有利。

回复
Arthur
2007年11月17日 09:05 编辑
我也有蛀牙，补了以后还有后遗症，冷热酸甜都会引起牙疼

回复
磊
2007年11月17日 00:04 编辑
天

回复
Qizhen
2007年11月16日 21:51 编辑
我使用 Google Reader 看别人blog的

回复
俊
2007年11月16日 19:48 编辑
每次都在outlook上看你的bolg，总是在点击打开文章页面回复滴时候页面慢吞吞的然后死翘翘了。靠。
 
想想自己写spaces用live writer，看spaces用outlook，确实很久灭登陆了。
 
今天终于登陆成功，在这里踩上一脚，鼓掌，噢耶～～～～～～～

回复
fang
2007年11月16日 18:29 编辑
有些恐怖的说。。。

回复
yoyo
2007年11月16日 18:25 编辑
跟我一样啊!

回复
